snippets/base/slack.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from django.conf import settings
+from django.template.loader import render_to_string
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+def send_slack(template_name, snippet):
+    data = render_to_string('slack/{}.jinja.json'.format(template_name),
+                            context={'snippet': snippet})
+    _send_slack(data)
+
+
+def _send_slack(data):
+    if not (settings.SLACK_ENABLE and settings.SLACK_WEBHOOK):
+        logger.info('Slack is not enabled.')
+        return
+
+    response = requests.post(settings.SLACK_WEBHOOK, data=data,
```

Comment:
Note: you could instead just build a `dict` and use `requests.post(url, json=data)` and it will `json.dumps` for you and set the proper content type. This works fine though if you just like using Jinja better.

Reply:
Aha didn't know about `json=`. I'll keep Jinja for the template functionality for this one but the requests trick is nice, thanks!